tools/internal_ci/macos/grpc_basictests_objc.cfg

```diff
@@ -29,3 +29,8 @@ env_vars {
   key: "RUN_TESTS_FLAGS"
   value: "-f basictests macos objc dbg --internal_ci -j 1 --inner_jobs 4 --bq_result_table aggregate_results"
 }
+
```

Comment:
looks like you're removing the "dbg" suffix from the job name, but you're still running only running "dbg" tests.
Is your plan to stop to stop running the "opt" tests completely? (that might not be wise, the dbg and opt versions of C core are quite different).

Reply:
The `dbg` and `opt` tags has been making no difference in how the ObjC stack is built in those tests because they both are testing against the same scheme. Currently the only difference it makes is how the plugin is built. We can potentially make these schemes different though. I'll check it out.